### PR TITLE
Add functions to allow programmatically adding of tooltip and position…

### PR DESCRIPTION
**Pull request by Tempus35**
I took two PR's that have not been merged #3004 and #2510 yet and added them and added ability to also send in tooltip text programmatically.

### CODE REVIEW

**Comment by acburst** [comment]
Thanks!

**Comment by wzup** [comment]
And how does it work? How to use? Were docs adjusted accordingly?

**Comment by Tempus35** [comment]
I do not have access to docs but it works as you had in you issues posts. You can now do:

$('#something').tooltip({delay:50,position:'top',tooltip:'Hello World'});

Instead of the data-tooltip etc on the element, also think there is no need for tooltipped class if you init yourself.

**Comment by GorgonsMaze** [comment]
@Tempus35 Do you know if you can change the text and background color using this? I'm trying  to change multiple tooltips throughout to different colors?

**Comment by Tempus35** [comment]
If you enable html via data-html="true" or html:true then you can do something like <p style="color:#000000">Something</p> for the text, at least that should work. As for background, I do not think that is possible at the moment. Might be good to make a new param that can take a custom class or classes.

File: js/tooltip.js~ [hunk 1]
```diff
@@ -0,0 +1,203 @@
+(function ($) {
```
Comment (annismckenzie) [inline js/tooltip.js~]: What's this file doing in this PR? Looks like a backup file of some sort.

Comment (Tempus35) [inline js/tooltip.js~]: Its likely something my IDE made, likely a backup that I forgot to exclude.

**Pull request merged**